hack/ci/e2e-apparmor.sh

```diff
@@ -25,7 +25,7 @@ APPARMOR_REFERENCE_TMP_PROFILE_FILE="/tmp/apparmorprofile-sleep-reference.yaml"
 APPARMOR_PROFILE_FILE_COMPLAIN_MODE="examples/apparmorprofile-sleep-complain-mode.yaml"
 SLEEP_INTERVAL_RECORDING="30"     # 30s sleep interval during recording.
 SLEEP_INTERVAL_VERIFICATION="300" # 5min to make sure that the enforcement check finds a running  PID.
-RUNTIMES=(runc crun)
+RUNTIMES=(crun runc)
```

Comment:
This shouldn't change anything (but is useful in https://github.com/kubernetes-sigs/security-profiles-operator/pull/2623 to make sure crun is not tainted by a previous runc test)